css/style.css

```diff
@@ -1,19 +1,205 @@
+/* Hero */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+.hero {
+    background: url("../img/first-background.jpg");
+    height: 650px;
+    background-position: right 27% bottom 5%;
```

Comment:
I would suggest adding "background-size: cover" this will ensure that the background doesn't repeat on wide screens